api/controllers/document.js

```diff
@@ -1,5 +1,5 @@
 var _ = require('lodash');
```

Comment:
Just wondering about tree shaking with lodash, and if you were making this a standard, or is there certain circumstances for this?

Reply:
Thanks for the review. I didn't need to do too much in this file so I forgot to have a look to update lodash to tree shake it. Thanks for flagging and I'll update the repo before merging.

Reply:
I'll actually handle this in a future PR. Thanks for flagging it